tests/scripts/all.sh

```diff
@@ -824,9 +824,9 @@ component_test_no_use_psa_crypto_full_cmake_asan() {
     # full minus MBEDTLS_USE_PSA_CRYPTO: run the same set of tests as basic-build-test.sh
     msg "build: cmake, full config + MBEDTLS_USE_PSA_CRYPTO, ASan"
```

Comment:
Change this message too, please.

Reply:
Change it to what?

Reply:
The "+" suggests that MBEDTLS_PSA_CRYPTO_C is set. There are a couple of conventions:

1. "full config + !MBEDTLS_USE_PSA_CRYPTO"
2. "full config without MBEDTLS_USE_PSA_CRYPTO"
3. "full config except MBEDTLS_USE_PSA_CRYPTO"
4. "full config + USE_PSA_REMOVED"

I would suggest 2 or 3.

Reply:
Oh, right. It's a preexisting bug. Fixing. Thanks.